odbc/src/api/functions.rs

```diff
@@ -350,10 +350,10 @@ pub extern "C" fn SQLColAttributeW(
         Desc::Searchable => numeric_col_attr(&|x: &MongoColMetadata| x.is_searchable as Len),
         Desc::TableName => string_col_attr(&|x: &MongoColMetadata| x.table_name.as_ref()),
         Desc::TypeName => string_col_attr(&|x: &MongoColMetadata| x.type_name.as_ref()),
+        Desc::Type => numeric_col_attr(&|x: &MongoColMetadata| x.sql_type.0 as Len),
```

Comment:
For now Type and ConciseType are identical, but I kept them as separate cases in case we decide to change this. I'd be up for just collapsing them for now too, what do you think?

Reply:
Let's collapse them for now; there are other places where we collapsed cases, so these 2 shouldn't be treated specially. We can separate them if/when we need to.